src/App.tsx

```diff
@@ -78,7 +87,14 @@ const routes =createBrowserRouter([
     path:"user",
     errorElement:<NotFound/>,
     children:[
-      {index:true,element:<LandingPage/>},
+      {index:true,element:<LandingPage />},
```

Comment:
We need to update the routing.
when we visit the base URL, we should be in the landing page not the login.

Reply:
We also need to have a user layout for the navigation and footer, and the landing and all related paths will be children of it. This is similar to other layouts that we have created.